state/actions.js

```diff
@@ -0,0 +1,17 @@
+import * as actionTypes from './actionTypes';
+
+// eslint-disable-next-line import/prefer-default-export
+export const sendMessage = (socketInstance, message) => (dispatch) => {
+  socketInstance.emit('message', message);
```

Comment:
@MagicWilliams this line essentially takes an instance of a socket connection and sends the message that the person typed in to the server using a websocket!